woocommerce/woocommerce-ios
# Readers details to support dynamic font size

Closes: #5521 


### Description

The PR replaces fixed size of the fonts and adjusts constraints so info about a connected reader readable with large font size

### Testing instructions

* Go to the settings -> IPP
* Connect to a reader
* Change the system font size to something big
* Notice that info about connected reader is readable now

### Screenshots

<img width="303" alt="image" src="https://user-images.githubusercontent.com/4923871/168589645-a45c0a37-0e0d-4d06-9c0d-44dd2bfe0f63.png">


---
